packages/quantic/force-app/main/default/lwc/quanticFeedbackModalQna/quanticFeedbackModalQna.js

```diff
@@ -0,0 +1,188 @@
+ *
+ * Under the hood, the component relies on a [`lightningModal`](https://developer.salesforce.com/docs/component-library/bundle/lightning-modal/documentation) component.
+ * For an example of how to use the `QuanticFeedbackModal` component, see the [`quanticSmartSnippet`](https://github.com/coveo/ui-kit/blob/master/packages/quantic/force-app/main/default/lwc/quanticSmartSnippet/quanticSmartSnippet.js) implementation.
+ *
+ * @category Search
+ * @category Insight Panel
+ */
+export default class QuanticFeedbackModalQna extends LightningModal {
+  labels = {
+    skip,
+    done,
+    sendFeedback,
+    helpUsImprove,
+    requiredFeedbackQuestion,
+    thankYouForYourFeedback,
+    answerEvaluation,
+    questionTopic,
+    questionHallucination,
+    questionDocumented,
+    questionReadable,
+    yourFeedbackHelps,
+    answerYes,
+    answerNo,
+    answerNotSure,
+    documentUrl,
+    documentUrlExplanation,
+    additionalNotes,
+    additionalNotesExplanation,
+  };
+
+  /**
+   * @api
+   * The function that will be executed when the feedback is submitted.
+   * @type {function}
+   */
+  @api handleSubmit;
+  /**
+   * @api
+   * If the feedback modal was opened on the thumbs up. (otherwise it was on a thumbs down).
+   * @type {boolean}
+   */
+  @api liked;
+
+  /** @type {boolean} */
+  isFeedbackSubmitted = false;
+  /** @type {GeneratedAnswerFeedbackOption} */
+  correctTopicValue;
+  /** @type {GeneratedAnswerFeedbackOption} */
+  hallucinationFreeValue;
+  /** @type {GeneratedAnswerFeedbackOption} */
+  documentedValue;
+  /** @type {GeneratedAnswerFeedbackOption} */
+  readableValue;
+  /** @type {string} */
+  documentUrlValue;
+  /** @type {string} */
+  detailsValue;
+
+  questions = [
+    {
+      question: this.labels.questionTopic,
+      changeFn: (event) => {
+        this.correctTopicValue = event.target.value;
+      },
+      id: 'correctTopic',
+    },
+    {
+      question: this.labels.questionHallucination,
+      changeFn: (event) => {
+        this.hallucinationFreeValue = event.target.value;
+      },
+      id: 'hallucinationFree',
+    },
+    {
+      question: this.labels.questionDocumented,
+      changeFn: (event) => {
+        this.documentedValue = event.target.value;
+      },
+      id: 'documented',
+    },
+    {
+      question: this.labels.questionReadable,
+      changeFn: (event) => {
+        this.readableValue = event.target.value;
+      },
+      id: 'readable',
+    },
+  ];
+
+  answerOptions = [
+    {label: this.labels.answerYes, value: 'yes'},
+    {label: this.labels.answerNotSure, value: 'unknown'},
+    {label: this.labels.answerNo, value: 'no'},
+  ];
+
+  /** @type {string} */
+  error;
+
+  /**
+   * Closes the modal.
+   * @returns {void}
+   */
+  closeModal() {
+    this.close();
+  }
+
+  get feedbackQuestionsRadios() {
+    return this.template.querySelectorAll('lightning-radio-group');
+  }
+
+  get feedbackFormIsValid() {
```

Comment:
Yea that's true I'll change it

Reply:
the `feedbackFormIsValue` is an actual property of the feedback form.

I could change it to a function as well and it would be perfectly fine to me as well.

That one was taken from an example in the salesforce doc, so it's more a property than a function for me.